Cargo.toml

```diff
@@ -70,10 +70,10 @@ pedantic = "warn"
 all = "warn"
 
 [workspace.dependencies]
-# stylus-related
-stylus-sdk = "0.6.0"
+# Stylus SDK related
+stylus-sdk = { version = "0.7.0", default-features = false, features = ["mini-alloc"] }
 
-alloy = { version = "=0.1.4", features = [
+alloy = { version = "=0.7.2", features = [
```

Comment:
I believe we should find a newer version of `alloy` that depends on `alloy-primitives` v0.8.14.